audio-worklet/basic/js/one-pole.js

```diff
@@ -0,0 +1,18 @@
+registerProcessor('one-pole', class OnePole extends AudioWorkletProcessor {
+  constructor () {
+    super();
+    let cutoffFrequency = 500;
+    this.b1 = Math.exp(-2 * Math.PI * cutoffFrequency / 48000);
+    this.a0 = 1.0 - this.b1;
+    this.z1 = 0;
+  }
+
+  process (input, output) {
+    let inputChannelData = input.getChannelData(0);
+    let outputChannelData = output.getChannelData(0);
+    for (let i = 0; i < 128; ++i) {
```

Comment:
Also, should this support more than one channel? (I thought the process method passed in arrays of arrays.)  As a demo, I guess supporting only mono is ok.